scripts/requirements.txt

```diff
@@ -26,3 +26,4 @@ pyOpenSSL==19.0.0
 ndg-httpsclient==0.5.1
 pyasn1==0.4.5
 hypothesis==4.24.3
+tensorflow==1.14.0
```

Comment:
Do we want to make TensorFlow a formal requirement yet?

Reply:
My bad. I don't want it to be a formal requirement yet.